R/learner_partykit_surv_cforest.R

```diff
@@ -133,21 +133,20 @@ LearnerSurvCForest = R6Class("LearnerSurvCForest",
       preds = invoke(predict, object = self$model, newdata = newdata,
         type = "prob", .args = pars)
 
-      # Define WeightedDiscrete distr6 distribution from the survival function
-      x = lapply(preds, function(z) {
-        time = c(0, z$time, max(z$time) + 1e-3)
-        surv = c(1, z$surv, 0)
-        data.frame(x = time, cdf = 1 - surv)
-      })
-      distr = distr6::VectorDistribution$new(
-        distribution = "WeightedDiscrete",
-        params       = x,
-        decorators   = c("CoreStatistics", "ExoticStatistics"))
+      times = lapply(preds, function(p) p$time)
```

Comment:
map_dbl

Reply:
Can't get it to work! `preds` is a list of objects for which you want to get the `$time` vector. using `map_dbl`:

Reply:
![image](https://github.com/mlr-org/mlr3extralearners/assets/10909555/c8e6a052-25b9-4e22-bb15-0c93ca9f6e46)

Reply:
Ah sorry yes this is what I thought